<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FreeBSD Config Tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/freebsd.jpg</normaloff>:/icons/icons/freebsd.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QMainWindow { background-color: #f5f7fa; }
    QPushButton {
        background-color: #4a90e2;
        color: white;
        border: none;
        padding: 14px 24px;
        border-radius: 10px;
        font-weight: bold;
        font-size: 13px;
        min-height: 44px;
    }
    QPushButton:hover { background-color: #5fa2f0; }
    QPushButton:pressed { background-color: #357abd; }
    QPushButton:disabled { background-color: #a0c4ff; opacity: 0.7; }
    QTextEdit {
        background-color: #1e1e1e;
        color: #d4d4d4;
        border: 1px solid #3a3a3a;
        border-radius: 12px;
        padding: 16px;
        font-family: &quot;Consolas&quot;, &quot;Fira Code&quot;, &quot;Monaco&quot;, monospace;
        font-size: 12px;
        selection-background-color: #4a90e2;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>24</number>
    </property>
    <property name="leftMargin">
     <number>24</number>
    </property>
    <property name="topMargin">
     <number>24</number>
    </property>
    <property name="rightMargin">
     <number>24</number>
    </property>
    <property name="bottomMargin">
     <number>24</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="leftButtonsLayout">
      <property name="spacing">
       <number>18</number>
      </property>
      <item>
       <widget class="QPushButton" name="cloneRepoButton">
        <property name="text">
         <string>  Клонировать ssd_log</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/icons/download.png</normaloff>:/icons/icons/download.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushRepoButton">
        <property name="text">
         <string>  Push ssd_log</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/icons/upload.png</normaloff>:/icons/icons/upload.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="buttonSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="extraButton1">
        <property name="text">
         <string>  Клонировать uname</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/icons/download.png</normaloff>:/icons/icons/download.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="extraButton2">
        <property name="text">
         <string>  Push uname</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/icons/upload.png</normaloff>:/icons/icons/upload.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="logTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Лог операций будет отображаться здесь...
Готов к работе.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">QStatusBar { color: #666; font-size: 11px; }</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
